source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_22.png
+        :width: 30 %
+
+    You are ready now to create a new account on testnet.
+
+    .. image:: ./images/wCCD_tutorial_19.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_20.png
+        :width: 30 %
+
+    You completed the browser wallet setup. Send some CCD to your new account or request some CCD from testnet faucet button within the browser wallet.
+    Check that your account balance is displayed and you have enough
+    CCD to be able to execute transactions.
+
+    .. note::
+        You are connected to a website with your browser wallet when you see the green ``Connected`` button.
+        You can toggle on/off the connection by clicking on the button.
+
+    .. image:: ./images/wCCD_tutorial_14.png
+        :width: 40 %
+
+Running the web front-end
+-------------------------
+
+You have successfully added the Concordium browser wallet extension to your browser in the previous section.
+In the next step of the tutorial, you are going to clone a wCCD demo front-end written with the React library
+and run it locally. The demo front-end has the required packages installed to connect to the Concordium browser wallet
+and implements common flows to deal with the different states that the browser wallet could be in. For example,
+the front-end will display a ``connect wallet`` button when loading the website. Furthermore, the front-end has flows
+to react to the events when the user switches the account in the browser wallet or
+connects/disconnects an account in the browser wallet to update the front-end state accordingly.
+
+.. note::
+
+    React is a popular open-source front-end JavaScript library.
+
+Clone this `repository <https://github.com/Concordium/concordium-browser-wallet>`_.
+
+.. code-block:: console
+
+    $git clone https://github.com/Concordium/concordium-browser-wallet.git
+
+Build and run the front-end as described in the README file of the
+`wCCD front-end demo <https://github.com/Concordium/concordium-browser-wallet/tree/main/examples/wCCD>`_.
+
+The steps in the README file are as follows:
+
+- Run ``yarn`` in the root folder to install all dependencies.
+
+.. code-block:: console
+
+    $yarn
+
+- Run ``yarn build:all`` to build the concordium-helpers package.
+
+.. code-block:: console
+
+    $yarn build:all
+
+- Navigate to the wCCD example folder.
+
+.. code-block:: console
+
+    $cd ./examples/wCCD/
+
+- Run ``yarn watch`` to enable hot-reload (useful for development) of the web front-end whenever you do any changes to the code.
+
+.. code-block:: console
+
+    $yarn watch
+
+- Run ``yarn start`` in another terminal to start the web front-end.
+
+.. code-block:: console
+
+    $yarn start
+
+This command logs a URL in the console (typically http://127.0.0.1:8080). Open this URL in the ``Chrome`` browser.
+
+.. note::
+
+    Check that your browser wallet is connected to the testnet and not to mainnet or stagenet.
```

Comment:
Is this very obvious? Or should we explain how that is done?

Reply:
Check that your browser wallet is connected to testnet (screenshots shown in workflow 1) or a testnet node (workflow 2) and not to mainnet or stagenet.